docs/gettingstarted/hardware.md

```
@@ -1,24 +1,37 @@
+* [Home control](https://www.myhome-control.de/), e.g. the ESP32-P4 Nano shield
+* [Athom](https://www.athom.tech/wled) / [Gledopto](https://www.gledopto.eu/Gledopto-WLED-LED-Controller-for-Digital-Light-Strips) / [M5Stack](https://shop.m5stack.com/collections/m5-controllers) / ...
+
+## DIY
+
+### Minimal hardware
+
+* One of the ESP32-devices shown in [MoonLight Installer](../installer). Any ESP32-S3 board or ESP32-P4 nano recommended as they have more memory (PSRAM) then the traditional ESP32-D0 boards.
 * Any WS2812 LED strip or panel. [256 LED panel](https://s.click.aliexpress.com/e/_EIKoYrg) recommended.
 * Connect the ESP32-device with USB to your computer, connect the LED strip or panel
 
 <img width="350" src="https://github.com/user-attachments/assets/1623a751-5f4b-463b-a6f3-a642c2bc52bf"/>
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)